react-app-drawer-component/my-app/src/AppDrawer.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactElement } from 'react';
+import DrawerItem from './DrawerItem';
+
+type AppDrawerProps = {
+  isOpen: boolean;
+  items: { id: number; name: string }[];
```

Comment:
You probably want to export a type for this so it can be used by callers of this component (such as when declaring `menuItems` in App.tsx).